README.md

```diff
@@ -18,4 +18,4 @@ The TRAMS (Trust and Academy Management System) system acts as a data source for
 
 The document is maintained here as a central contract so systems can develop against a consistent API while it is in development. It's generated and viewable using (Swagger)[swagger.io]
 
-Use `docker run -d -p 80:8080 -v $(pwd):/tmp -e SWAGGER_FILE=/tmp/trams_api_spec.yaml swaggerapi/swagger-editor` to run the editor with the trams api spec pre loaded, on localhost:80
+Use `docker run -d -p 80:8080 -v $(pwd):/tmp -e SWAGGER_FILE=/tmp/trams_api_spec/trams_api_spec.yaml swaggerapi/swagger-editor` to run the editor with the trams api spec pre loaded, on localhost:80
```

Comment:
one day we'll get this command right - that /tmp/ folder is in the docker image so shouldn't need a directory?

Reply:
You're mounting the root onto `/tmp`, you mount the entire project. Inside docker means you need the directory. Or you mount the directory where you define the volume - depends where you're running docker. I was running it in the root of the project so I didn't need it 

Maybe a Make command or simliar

Reply:
Yeah depends on where you run the command from, a make command is a good idea